tests/model/VirtualPageTest.php

```diff
@@ -200,51 +198,12 @@ public function testCantPublishVirtualPagesBeforeTheirSource()
         $this->assertTrue($vp->canPublish());
     }
 
-    public function testCanDeleteOrphanedVirtualPagesFromLive()
```

Comment:
ooh, no more orphaned virtual pages, no wonder it's no longer needed in the "broken links" report